### PR TITLE
refactor: correctly sort wallets and contacts in overview and sidebar

### PR DESCRIPTION
**Pull request by dated**
## Proposed changes

This fixes the sorting by name of the wallets and contacts in the grid view, table view and sidebar.

It also changes the implementation of the `sortByProp` (now `sortByProps`) util, which now uses [natural sorting](https://en.wikipedia.org/wiki/Natural_sort_order) by default and allows for multiple properties.

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] Refactoring (improve a current implementation without adding a new feature or fixing a bug)

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes

### CODE REVIEW

**Comment by dated** [comment]
As discussed with Alex, wallets without a name (custom or from the Network) will stay at the top of the list.

**Comment by dated** [comment]
Does that happen with these changes @zillionn ?

**Comment by zillionn** [comment]
No, they're not.

**Comment by dated** [comment]
Thanks for the feedback, I'll take a look asap!

**Comment by dated** [comment]
should be fixed now @zillionn, stupid typo 😉

**Comment by dated** [comment]
Done as requested @j-a-m-l

**Comment by dated** [comment]
Better?

**Comment by j-a-m-l** [comment]
Thanks @dated. Now it's a lot easier to read and it's also easier to grasp what should be expected when sorting.

**Comment by j-a-m-l** [comment]
@dated I just realized that the Ledger wallets are also visible when navigating through the contacts page, so the sidebar is showing both, contacts and owned Ledger wallets. Would you mind to change it?

**Comment by dated** [comment]
I've made some changes to how the wallets are fetched, I can't test it myself right now unfortunately.

File: src/renderer/components/Wallet/WalletSidebar/WalletSidebar.vue [hunk 1]
```diff
@@ -195,11 +195,14 @@ export default {
   }),
 
   computed: {
+    fetchContactsOnly () {
```
Comment (j-a-m-l) [inline src/renderer/components/Wallet/WalletSidebar/WalletSidebar.vue]: I'd rename it to `hasContactsOnly`, `includesContactsOnly` or something like that, using "fetch" looks like that method / computed property request the data.

Comment (dated) [reply src/renderer/components/Wallet/WalletSidebar/WalletSidebar.vue]: i changed it to `hasContactsOnly`

**Pull request merged**